network_testing/client_server_genhtml.py

```diff
@@ -34,28 +33,10 @@ def load_data(filename):
     else:
         raise ValueError('Unknown extension: ' + filename)
 
-@click.command(help=__doc__)
-@click.option('-s', '--staticdatadir', default=os.path.join(data_path, 'report/static_data/'),
-              help='Alternate directory containing static data such as test descriptions')
-@click.option('-o', '--outdir', default=os.path.join('.', 'html-output/'),
-              help='Directory for output (will be overwritten)')
-@click.option('--templatedir', default=os.path.join(data_path, 'report/templates/'),
-              help='Directory containing HTML templates')
-@click.argument('input_file', nargs=-1)
 def build(staticdatadir, outdir, templatedir, input_file):
-    if not input_file:
-        raise click.UsageError('No files given. Try running: {} {}'.format(
-              sys.argv[0], os.path.join(data_path, 'example_data/*')))
-        print(__doc__, file=sys.stderr)
-        print(file=sys.stderr)
-        print('No files given. Try running:',
-              sys.argv[0], os.path.join(data_path, 'example_data/*'),
-              file=sys.stderr)
-        return
-
     try:
         shutil.rmtree(outdir)
-    except FileNotFoundError:
+    except Exception:
```

Comment:
I did not think about that much, it was caused by missing FileNotFoundError exception in python2.7 on Fedora23 so that I had to replace it by something ahother. If OSError is there then it is better candidate that general Exception